src/sentry/profiles/task.py

```diff
@@ -204,11 +197,76 @@ def _symbolicate(profile: Profile, project: Project) -> None:
         except Exception as e:
             sentry_sdk.capture_exception(e)
             break
+    return []
```

Comment:
If we ever reach this line, then `_process_symbolicator_results_for_sample` function will error since it assumes an array of 1 element for the sampled format. Can we return the original `stacktraces` and pass it through unsymbolicated?